tests/data/zuericrop/data.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import h5py
+from pathlib import Path
+
+import numpy as np
+
+
+# Sentinel-2 is 12-bit with range 0-4095
+SENTINEL2_MAX = 4096
+
+NUM_SAMPLES = 2
+NUM_CHANNELS = 9
+SIZE = 24
+NUM_CLASSES = 10
+
+np.random.seed(0)
+
+data_file = "ZueriCrop.hdf5"
+labels_file = "labels.csv"
+
+# Remove old data
+os.remove(data_file)
+os.remove(labels_file)

```

Comment:
I would guard these with `if os.path.exists(...):` so that this script works even if the files aren't already present.

Reply:
```suggestion
if os.path.exists(data_file):
    os.remove(data_file)
if os.path.exists(labels_file):
    os.remove(labels_file)
```